app/eth2wrap/eth2wrap.go

```diff
@@ -57,7 +57,21 @@ func Instrument(clients ...Client) (Client, error) {
 		return nil, errors.New("clients empty")
 	}
 
-	return newMulti(clients), nil
+	// TODO(gsora): remove once the implementation is agreed upon and
```

Comment:
Is this still valid?

Reply:
Yes, the code does not have any wiring to the CLI interface, hence it acts as if it was a non-fallback client.